videos/videos.js

```diff
@@ -144,7 +157,7 @@ router.post("/", restricted, async (req, res) => {
     }
   } catch (err) {
     console.log(err);
-    res.status(500).json({ message: "Try again later!", err });
+    res.status(500).json({ message: "Try again later 3", err });
```

Comment:
Ill come back on the next round and update the error messages to return more friendly and be relevant to the route called.
Nice job hooking up the S3 bucket